src/components/z-menu/index.tsx

```diff
@@ -77,6 +86,71 @@ export class ZMenu {
     this.closed.emit();
   }
 
+  @Listen("keydown")
+  handleKeyDown(e: KeyboardEvent): void {
+    if (e.code === KeyboardCode.ENTER) {
+      return;
+    }
+    this.handleArrowsNav(e);
+  }
+
+  private handleArrowsNav(e: KeyboardEvent): void {
```

Comment:
questa tipologia di metodo, siccome viene spesso usato in diversi componenti, non potrebbe diventare una utils?

Reply:
scarterei l'idea di utilizzarla come utilis, la navigazione non è uguale per tutti ed è poi specifica per il componente.